rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/edition/InlineItemModificationCreator.java

```diff
@@ -44,123 +45,67 @@
 
     public static final String EDIT_FORM_ITEM_SUFFIX = ".edit";
 
-    public static final String EDIT_RADIO_OPTION_NAME = "edit";
+    private static final String EDIT_RADIO_OPTION_NAME = "edit";
 
-    public static final String UPLOAD_RADIO_OPTION_NAME = "upload";
+    private static final String UPLOAD_RADIO_OPTION_NAME = "upload";
 
     private final NodeSourceWindow nodeSourceWindow;
 
+    private FormItem inlineEditingFormItem;
+
+    private FormItem chooseCredentialsFormItem;
+
+    private RadioGroupItem editOrUploadFormItem;
+
     public InlineItemModificationCreator(NodeSourceWindow nodeSourceWindow) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)